scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -110,6 +110,35 @@ def main():
     """
     global GITHUB_TOKEN
 
+    # The script runs as
+    # .../microshift/scripts/release-notes/gen_ec_release_notes.py
+    # and we want the path to
+    # .../microshift
+    root_dir = pathlib.Path(__file__).parent.parent.parent
+    version_makefile = root_dir / 'Makefile.version.aarch64.var'
```

Comment:
Can't we assume we're executing the script from the `microshift/` and just use current working dir?
That's how it runs in the https://github.com/openshift/release/pull/49830/files#diff-24ec5dd51fc1cbf37ac7d08f85d9f4d8abc188083abce1ccc7ace59d1eaf6377R16-R17